# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/chrome-webidl/moz.build
# Commit: 65cc1910918c
# Full Hash: 65cc1910918cf410ada326fd353db1da51c2c0eb
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-12-07 21:55:34
# Regressor Bug: 1588839
# File Overlap Count: 1
# Description:
#   Bug 1588839 - Part 1 - Add helper to allow structured cloning Error/Exception objects. r=bzbarsky
#   
#   In order to be able to reasonably debug error results from things like
#   JSWindowActor.sendQuery, we need to be able to clone errors across process
#   boundaries, so that they can be propagated to the caller that initiated a
# ==============================================================================

diff -r 2f8b5b48c896 -r 65cc1910918c dom/chrome-webidl/moz.build
--- a/dom/chrome-webidl/moz.build	Mon Dec 02 18:28:28 2019 +0000
+++ b/dom/chrome-webidl/moz.build	Sat Dec 07 04:43:24 2019 +0000
@@ -35,6 +35,7 @@
 WEBIDL_FILES = [
     'BrowsingContext.webidl',
     'ChannelWrapper.webidl',
+    'ClonedErrorHolder.webidl',
     'DebuggerNotification.webidl',
     'DebuggerNotificationObserver.webidl',
     'DebuggerUtils.webidl',